vendor/engines/secure_rooms/spec/features/scans_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe "Scanning in and out", type: :controller do
```

Comment:
There's already a `scans_controller_spec` there and this is testing a very different layer than what's currently in there. Would calling it something like `scans_controller_feature_spec.rb` be too weird? Or maybe it's better just to put it in that same file?